infra/recipes/k8s/helm/tables/config/azure/sandbox/cluster.yaml

```diff
@@ -2,14 +2,23 @@ cluster:
   name: "AKSCluster"
   storage:
     type: "hadoop"
-    uri: "file:///"
-    root-path: "/tmp/openhouse"
+    uri: "hdfs://namenode:9000/"
+    root-path: "/data/openhouse"
+  storages:
+    default-type: "adls"
+    types:
+      adls:
+        rootpath: <container-name>@<storage-account-name>.dfs.core.windows.net 
+        endpoint: "abfs://"
+        parameters:
+          adls.auth.shared-key.account.name: <storage-account-name>
+          adls.auth.shared-key.account.key: <storage-account-key>
```

Comment:
Nit: Is this updated in README.

Reply:
Ill make sure it is

Reply:
Actually nevermind no action is needed from the user, it is all replaced automatically by the helm templates